app/main/routes.py

```diff
@@ -733,48 +730,31 @@ def download_record(record_id: uuid.UUID):
 
     validate_body_user_groups_or_404(file.consignment.series.body.Name)
 
-    s3 = boto3.client("s3")
     bucket = current_app.config["RECORD_BUCKET_NAME"]
     key = f"{file.consignment.ConsignmentReference}/{file.FileId}"
 
-    try:
-        s3_file_object = s3.get_object(Bucket=bucket, Key=key)
-    except Exception as e:
-        current_app.app_logger.error(f"Failed to get object from S3: {e}")
-        abort(404)
-
     download_filename = file.FileName
-
     if file.CiteableReference:
         if len(file.FileName.rsplit(".", 1)) > 1:
             download_filename = (
                 file.CiteableReference + "." + file.FileName.rsplit(".", 1)[1]
             )
 
     try:
-        file_content = s3_file_object["Body"].read()
-        file_type = download_filename.split(".")[-1].lower()
+        presigned_url = s3.generate_presigned_url(
+            "get_object",
+            Params={
+                "Bucket": bucket,
+                "Key": key,
+                "ResponseContentDisposition": f"attachment; filename={download_filename}",
+            },
+            ExpiresIn=3600,
+        )
     except Exception as e:
-        current_app.app_logger.error(f"Error reading S3 file content: {e}")
+        current_app.app_logger.error(f"Failed to generate presigned URL: {e}")
         abort(500)
 
-    content_type = s3_file_object.get("ContentType", "application/octet-stream")
-
-    if render:
```

Comment:
This condition block was already redundant anyway wasn't in, since we moved render to use presigned urls directly.